templates/inspec/iam_policy/iam_policy.md.erb

```diff
@@ -40,6 +40,10 @@ Properties that can be accessed from the `<%= iam_resource_name -%>` resource:
 
     * `audit_log_configs`: The configuration for logging of each type of permission.
 
+      * `log_type`: The log type that this config enables. For example, ADMIN_READ, DATA_WRITE or DATA_READ
```

Comment:
Why are these docs hand-written?

Reply:
These docs are shared between all IAM policies as they describe the structure of the IAM policy that is shared across many resources. 

They are hand written because they *should* be relatively static, and it allows us to have a template for IAM policy docs that we can add resource specific info to.